### PR TITLE
Added Wishlist Page and linked to navbar

### PR DESCRIPTION
**Pull request by rutu14**
To access the Wishlist page click on the heart icon on navbar.

On hovering over the card, move to card button appears.

### CODE REVIEW

File: css/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+.heading-icon{
+    font-size: 24px;
```
Comment (zoef-yt) [inline css/wishlist.css]: Use rem for fontsize.

File: css/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    transform: translate(14rem, 10px);
+    position: absolute;
+    border: none;
+    padding: unset;
+}
+
+.wishlist-icon{
+    font-size: 2.7rem;
+    color: var(--primary-colour);
+    filter: drop-shadow(0.65rem 0.35rem 0.4rem var(--wishlist-shadow));
```
Comment (zoef-yt) [inline css/wishlist.css:21]: px can be used here.

File: wishlist.html [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+                            <button class="button-icon nav-icon cp">
+                                <i class="bi bi-heart"></i>
+                                <span class="textButtonBadge nav-icon-badge">9</span>
+                            </button>
+                        </a>
+                    </li>
+                </ul>
+            </nav>
+            <div class="account">  
+                <!-- <a href="login-page.html" class="btn btn-primary outline  td login-sign" type="button">Login</a> -->
```
Comment (zoef-yt) [inline wishlist.html:46]: Remove commented code.

**Review by zoef-yt — APPROVED** [review]
The code looks good. Ui looks amazing. Good to merge. Good job.

File: wishlist.html [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+    <main>
+        <h3 class="heading2 medium text-center m-t40">Captured in your 
+            <svg xmlns="http://www.w3.org/2000/svg" width="40" vertical-align="text-top" height="40" fill="#7f1a78" class="bi bi-bag-heart" viewBox="0 0 16 16">
+                <path fill-rule="evenodd" d="M10.5 3.5a2.5 2.5 0 0 0-5 0V4h5v-.5Zm1 0V4H15v10a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V4h3.5v-.5a3.5 3.5 0 1 1 7 0ZM14 14V5H2v9a1 1 0 0 0 1 1h10a1 1 0 0 0 1-1ZM8 7.993c1.664-1.711 5.825 1.283 0 5.132-5.825-3.85-1.664-6.843 0-5.132Z"/>
+            </svg>
+        </h3>
+        <section class="wishlist-grid grid-4-cols"> 
+            <div class="card card-box-shadow wishlist-card">
+                <img class="card-img card-img-border product-card-img" src="/assests/images/product-page/clothing (1).jfif">
+                <button class="btn wishlist-icon-btn cp">
```
Comment (bhushannaikaroskar) [inline wishlist.html]: Use alt attribute in all images.

**Review by bhushannaikaroskar — CHANGES_REQUESTED** [review]
please use alt attribute in all the  images its a good practice

**Review by Omkar-Ghate — APPROVED** [review]
Great UI. Looks good to merge.

File: css/wishlist.css (hunk 2 above)
Comment (rutu14) [reply css/wishlist.css:21]: Using px is not giving the desired results for now, will change it in future.

File: wishlist.html (hunk 3 above)
Comment (rutu14) [reply wishlist.html:46]: Ok

File: wishlist.html (hunk 4 above)
Comment (rutu14) [reply wishlist.html]: Thanks for reminding, will change.

**Review by bhushannaikaroskar — APPROVED** [review]
Good to merge now

**Pull request merged**